src/components/ContextBar.js

```diff
@@ -19,50 +25,77 @@ const contextBarStyles = {
+  const terminalLaunchLink = Nav.getLink('workspace-application-launch', { namespace, name: workspaceName, application: 'terminal' })
+
+  const startCurrentRuntime = () => {
+    const { googleProject, runtimeName } = currentRuntime
+    Ajax().Runtimes.runtime(googleProject, runtimeName).start()
+  }
+
+  return h(Fragment, [
+    h(CloudEnvironmentModal, {
+      isOpen: isCloudEnvOpen,
+      onSuccess: () => {}, //TODO: I don't know if we want to do anything?
+      onDismiss: () => setCloudEnvOpen(false),
+      runtimes, apps, galaxyDataDisks, refreshRuntimes, refreshApps,
+      workspace,
+      canCompute,
+      persistentDisks
+    }),
+    div({ style: Style.elements.contextBarContainer }, [
+      div({ style: contextBarStyles.contextBarContainer }, [
+        h(MenuTrigger, {
+          closeOnClick: true,
+          content: h(Fragment, [
+            h(MenuButton, { onClick: () => setCloningWorkspace(true) }, [makeMenuIcon('copy'), 'Clone']),
+            h(MenuButton, {
+              disabled: !canShare,
+              tooltip: !canShare && 'You have not been granted permission to share this workspace',
+              tooltipSide: 'left',
+              onClick: () => setSharingWorkspace(true)
+            }, [makeMenuIcon('share'), 'Share']),
+            h(MenuButton, { disabled: true }, [makeMenuIcon('export'), 'Publish', comingSoon]),
+            h(MenuButton, {
+              disabled: !isOwner,
+              tooltip: !isOwner && 'You must be an owner of this workspace or the underlying billing project',
+              tooltipSide: 'left',
+              onClick: () => setDeletingWorkspace(true)
+            }, [makeMenuIcon('trash'), 'Delete Workspace'])
+          ]),
+          side: 'bottom'
+        }, [
+          h(Clickable, {
+            'aria-label': 'Menu',
+            style: contextBarStyles.contextBarButton,
+            hover: { boxShadow: `inset -6px 0px ${terraSpecial(0.9)}` },
+            tooltip: 'Menu',
+            tooltipDelay: 100
+          }, [icon('ellipsis-v', { size: 24 })])
+        ]),
+        h(Clickable, {
+          style: contextBarStyles.contextBarButton,
+          hover: { boxShadow: `inset -6px 0px ${terraSpecial(0.9)}` },
+          onClick: () => setCloudEnvOpen(!isCloudEnvOpen),
+          ...{ tooltip: 'Compute Configuration', tooltipDelay: 100 },
+          'aria-label': 'Compute Configuration'
+        }, [icon('cloudBolt', { size: 24 })]),
+        h(Clickable, {
+          style: { ...contextBarStyles.contextBarButton, color: terminalDisabled ? colors.dark(0.7) : contextBarStyles.contextBarButton.color },
+          hover: { boxShadow: `inset -6px 0px ${terraSpecial(0.9)}` },
+          disabled: terminalDisabled,
+          href: terminalLaunchLink,
+          onClick: window.location.hash === terminalLaunchLink && currentRuntime?.status === 'Stopped' ? () => startCurrentRuntime() : undefined,
```

Comment:
the terminal will start the runtime if its stopped. Users will not be able to click on this if the terminal is not in some other ready state.